src/cnxml/deserialize.js

```diff
@@ -182,6 +182,21 @@ const MARK_TAGS = {
 }
 
 
+/**
+ * Process data for captions.
+ */
+function caption(el, next) {
+    let type = 'caption'
+    if (el.parentElement) {
+        type = el.parentElement.tagName.toLowerCase() + '_caption'
+    }
```

Comment:
Don't do that. You have no guarantee that CNXML element names will match Slate blocks one-to-one (e.g. `<note>` vs `admonition`), and no guarantee that future elements with captions will also want to deserialize them this way. Just ignore `<captions>` which are not inside a `<figure>`.